src/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { router } = require('./routes/artist');
+const { albumRouter } = require('./routes/album');
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', router);
+app.use('/album', albumRouter);
+
+
+module.exports = app;
```

Comment:
You never need more than one clear line in your code, one of these should be removed
```
app.use('/album', albumRouter);

module.exports = app;
```